provider/pkg/internal/pulumiapi/webhooks.go

```diff
@@ -30,24 +30,20 @@ type Webhook struct {
 }
 
 type CreateWebhookRequest struct {
```

Comment:
Nit: I'd suggest renaming this `WebhookRequest` now that it's not technically just for create.